src/App.js

```diff
@@ -1,24 +1,55 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Routes, Route, NavLink } from 'react-router-dom';
 import './App.css';
 
+function List() {
+  return (
+    <>
+      <h1>List of Items</h1>
+    </>
+  );
+}
+
+function AddAnItem() {
+  return (
+    <>
+      <h1>Add an Item</h1>
+    </>
+  );
+}
+
+function NavLinks() {
+  return (
+    <nav style={{ position: 'fixed', bottom: 0 }}>
+      <NavLink
+        to="/"
+        style={({ isActive }) =>
+          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
+        }
+      >
+        List of Items
+      </NavLink>
+      <NavLink
+        to="/add-an-item"
+        style={({ isActive }) =>
+          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
+        }
+      >
+        Add an Item
+      </NavLink>
+    </nav>
+  );
+}
+
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Welcome to React Router!</h1>
```

Comment:
Maybe we want to say something more along the lines of "Welcome to your Smart Shopping List!"?

Reply:
Thank you for the review! We changed the title.